projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts

```diff
@@ -53,7 +60,7 @@ context("Tabbing order - tests don't require user to be logged in", () => {
     });
 
     it('should allow to navigate with tab key (filled out form)', () => {
-      loginTabbingOrder(config.login, true);
+      loginTabbingOrder(config.login);
```

Comment:
why was it removed?

Reply:
Doesn't change anything now that we have the login button patched.

Reply:
if so, please remove that additional boolean parameter from `loginTabbingOrder()` function